CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Features (non-breaking)
 * (precompiles) Add slinky precompiled contract
+* (x/warden) Add broadcastType to SignRequest
+* (precompiles) Add broadcastType to SignRequest
```

Comment:
these are actually breaking changes (since the old version of the node won't be able to handle the new transactions). In fact, the slinky precompile contract is also a breaking change (i know you are not the one who added it). Let's move them all under a section called `### Consensus Breaking Changes` like we used to do before, otherwise we'll risk to halt our chain like it happened last week :)

we don't have a written guide on how to decide whether a change is breaking or not, but i'll work on it

Reply:
done